src/main/java/com/gary/backendv2/model/dto/request/FacilityRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.gary.backendv2.model.dto.request;
+
+import com.gary.backendv2.model.enums.FacilityType;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+public class FacilityRequest {
+	@NotNull
+	private Double longitude;
+	@NotNull
+	private Double latitude;
+	@NotBlank
+	private String name;
+	@NotBlank
+	private FacilityType facilityType;
```

Comment:
`@NotBlank` -> `@NotNull`

or write a custom validator for enum types, because the only standard annotations which can be applied to enums are `@NotNull` and `@Null`.